demos/README.md

```diff
@@ -248,6 +248,7 @@ list above.
 ### Get Ready for Running the Demo Applications on Windows*
 
 Before running compiled binary files, make sure your application can find the Inference Engine and OpenCV libraries.
+Optionally download OpenCV community FFmpeg plugin. There is a downloader script in the OpenVINO package: `opencv\ffmpeg-download.ps1`.
```

Comment:
> Optionally download OpenCV community FFmpeg plugin
Not clear, why user should do this.
> There is a downloader script in the OpenVINO package: `opencv\ffmpeg-download.ps1`.
I think, it will be better `<INSTALL_DIR>\opencv\ffmpeg-download.ps1`

Reply:
I can list only two reasons: one wants opencv to use ffmpeg backend (obvious, no need to clarify). And one wants to read an image with VideoCapture which is actually sub point for the first reason. But this approach is discouraged by Alexander Alekhin and a common module for reading inputs should use imread() instead. Following Alexander Alekhin we don't what to encourage others to use ffmpeg with videocapture to read a single image.

I don't know how to clarify why a user may want to do this.

> I think, it will be better <INSTALL_DIR>\opencv\ffmpeg-download.ps1

It's clear because of `in the OpenVINO package`.

Reply:
> I can list only two reasons: one wants opencv to use ffmpeg backend (obvious, no need to clarify).

That is indeed obvious, but that just shifts the question: why would the user want to use the FFmpeg backend? For someone who is not well-acquainted with the various OpenCV backends, it won't be clear whether they want it or not. 

> It's clear because of in the OpenVINO package.

That may be so, but the document already uses `<INSTALL_DIR>` in various places. It would be consistent to use it here, too.

Reply:
As i said, I don't know how to clarify why a user may want to do this. Do you have an idea how to reformulate the note and clarify the usage of ffmpeg backend?

> That may be so, but the document already uses <INSTALL_DIR> in various places. It would be consistent to use it here, too.

Ok.